### PR TITLE
progressbar: Corrections after review

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

File: common.blocks/progressbar/progressbar.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ provide(BEMDOM.decl(this.name, /** @lends progressbar.prototype */{
      * @param {Number} value
      */
     setVal : function(value) {
-        this._setBarWidth(this._val = value);
+        this.elem('bar').css('width', (this._val = value) + '%');
```
Comment (persidskiy) [inline common.blocks/progressbar/progressbar.js:27]: why wouldn't move `this._val = value` to separate line?

**Comment by persidskiy** [comment]
Why do you check [private fields](https://github.com/bem/bem-components/blob/v2/common.blocks/progressbar/progressbar.spec.js#L24) in specs?

**Comment by persidskiy** [comment]
`setVal` doesn't return `this` in `_theme_simple` (https://github.com/bem/bem-components/blob/v2/design/common.blocks/progressbar/_theme/progressbar_theme_simple.js#L20)
also there is no need to write documentation for `@override`d metrods

File: common.blocks/progressbar/progressbar.js (hunk 1 above)
Comment (gela-d) [inline common.blocks/progressbar/progressbar.js:27]: @Coltspb why we should? :)
In this case we have minus one line.

**Comment by gela-d** [comment]
> Why do you check private fields in specs?

@Coltspb Thanks, fixed.

> also there is no need to write documentation for @overrided metrods

Our method add some functionality in `progressbar_theme_simple.js`, thats why I want to write documentation for `@override` method

File: common.blocks/progressbar/progressbar.spec.js [hunk 2]
```diff
@@ -21,21 +21,18 @@ describe('progressbar', function() {
             var value = 15;
             progressbar.setVal(value);
             progressbar.elem('bar')[0].style.width.should.be.equal(value + '%');
-            progressbar._val.should.be.equal(value);
         });
 
         it('should set correct percent if value is string with numbers', function() {
```
Comment (narqo) [inline common.blocks/progressbar/progressbar.spec.js]: This test contravenes block's API from jsdoc: `Progressbar#setVal(val {Number})`. Why we care about string behaviour?

File: design/common.blocks/progressbar/_theme/progressbar_theme_simple.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ provide(Progressbar.decl({ modName : 'theme', modVal : 'simple' }, /** @lends pr
     setVal : function() {
         this.__base.apply(this, arguments);
```
Comment (narqo) [inline design/common.blocks/progressbar/_theme/progressbar_theme_simple.js]: You could write it in the way below, to be more accurate:

``` js
var res = this.__base.apply(this, arguments);
this.elem('text').text(this._val);
return res;
```

**Comment by gela-d** [comment]
@narqo done

**Comment by narqo** [comment]
looks good

**Pull request merged**